app/assets/javascripts/components/ProjectSelection.jsx

```diff
@@ -58,6 +58,7 @@
       })
       .then((res) => {
         Samples.hideLoader();
+        favStatus == 'true' ?  _satellite.track('unfavorite') : _satellite.track('favorite');
```

Comment:
why is favStatus true is actually unfavorite? it's uninuitive. 
Also, I think tracking should be move to before the ajax calls. We want to track the intent. Not necessarily after it's successful.

Reply:
okay